docs/index.md

```diff
@@ -89,21 +89,31 @@ Don't worry, call `finetuner.describe_models()` and we will help you choose the
 :caption: How it Works
 :hidden:
 
-1_how_it_works/index
+1_how_it_works/1_1_how_it_works.md
+1_how_it_works/1_2_difference.md
 ```
 
 ```{toctree}
-:caption: Step By Step
+:caption: Get Started
 :hidden:
 
-2_step_by_step/index
+2_step_by_step/2_1_install.md
+2_step_by_step/2_2_experiment_and_run.md
+2_step_by_step/2_3_login_to_jina_ecosystem.md
+2_step_by_step/2_4_create_training_data.md
+2_step_by_step/2_5_choose_backbone.md
+2_step_by_step/2_6_start_finetune.md
+2_step_by_step/2_7_get_tuned_model.md
+2_step_by_step/2_8_next_steps.md
 ```
 
 ```{toctree}
-:caption: Finetuner in Action
+:caption: User Guides
```

Comment:
I think these should be called 'How-To' in order to better align with core docs

Reply:
Good point!